src/main/java/ca/on/oicr/pineryreports/util/GeneralUtils.java

```diff
@@ -16,6 +18,15 @@ private GeneralUtils() {
     throw new IllegalStateException("Util class not intended for instantiation");
   }
 
+  public static final Option OPT_AFTER = CommonOptions.after(false);
+  public static final Option OPT_BEFORE = CommonOptions.before(false);
+  public static final Option OPT_USER_IDS = CommonOptions.users(false);
+  public static final Option OPT_PROJECT = CommonOptions.project(true);
```

Comment:
Options aren't immutable, so sharing them statically can cause unexpected behaviour if one class modifies the object for some reason. Since a program execution is limited to one report, this may not be a problem, but probably still better to avoid. Also not clear which are required.